parsl/executors/high_throughput/interchange.py

```diff
@@ -411,11 +415,12 @@ def process_task_outgoing_incoming(self, interesting_managers: Set[bytes], hub_c
                     # We set up an entry only if registration works correctly
                     self._ready_managers[manager_id] = {'last_heartbeat': time.time(),
                                                         'idle_since': time.time(),
-                                                        'block_id': None,
+                                                        'block_id': msg['block_id'],
```

Comment:
this already happens on line 423/428: `m.update(msg)` I think? because it copies all the fields across from the registration message into the ready manager record. I don't think the pattern of "initialise everything blank here, then update it 10 lines later" needs to be specially different for the block id compared to the other fields in the manager record.

Reply:
This change is unrelated to what is going on in this PR anyway, so I'll remove this.